scss/_utilities_grid.scss

```diff
@@ -0,0 +1,19 @@
+@mixin grid-first-col {
```

Comment:
Couldn't we just add sizes to `u-no-margin` and `u-no-padding`. These classes are very specific.

Reply:
I agree that they're very specific to the pattern but is that a bad thing. 

Using utility classes to fix a pattern isn't a good plan, the pattern should work with its own BEM classes. We should discourage the use of utilities unless they're absolutely necessary. This is a tricky use-case as they're workaround classes for a tricky problem, so it's not quite so black-and-white but I, personally, prefer this to using utilities.